package.json

```diff
@@ -2,10 +2,12 @@
   "name": "angularsdk",
   "version": "0.0.0",
   "scripts": {
-    "build:prod": "npm run prod-build-angularsdk",
+    "build:prod": "npm run lint && npm run prod-build-angularsdk",
```

Comment:
You may want to make this work more like what I did in SDK-R with using the run-p package to allow linting and building to be done in parallel. It will make the experience be much faster.

Reply:
Done.